javascript/javascript1/week3/seriesDuration.js

```diff
@@ -0,0 +1,36 @@
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 3,
+    hours: 1,
+    minutes: 0,
+  },
+  {
+    title: "Prison Break",
+    days: 2,
+    hours: 18,
+    minutes: 0,
+  },
+  {
+    title: "The Wire",
+    days: 2,
+    hours: 12,
+    minutes: 0,
+  },
+];
+
+function logOutSeriesText() {
+  let totalTime = 0;
+  const lifespanInHours = 80 * 365 * 24;
+  for (let item of seriesDurations) {
+    const seriesTimeHours = item.days * 24 + item.hours + item.minutes / 60;
+    let percentage = seriesTimeHours / lifespanInHours;
```

Comment:
You forgot to * 100 to get the percentage.

```suggestion
    let percentage = seriesTimeHours / lifespanInHours * 100;
```